docs/src/pages/components/tables/ReactVirtualizedTable.js

```diff
@@ -17,8 +17,12 @@ const styles = (theme) => ({
     // temporary right-to-left patch, waiting for
     // https://github.com/bvaughn/react-virtualized/issues/454
     '& .ReactVirtualized__Table__headerRow': {
-      flip: false,
-      paddingRight: theme.direction === 'rtl' ? '0 !important' : undefined,
```

Comment:
Wasn't that already fine? Just the `flip: flase` is unused I thought.

Reply:
Not really, here is the result:
- master:

LTR:
```
paddingRight: undefined,
```
RTL:
```
paddingRight: '0 !important',
```
- next (without the changes on the PR):

LTR (correct):
```
paddingRight: undefined,
```
RTL:
```diff
-paddingRight: '0 !important',
+paddingLeft:  '0 !important',
```